src/shared/creds-util.c

```diff
@@ -253,6 +254,17 @@ int read_credential_strings_many_internal(
         return ret;
 }
 
+int read_credential_bool(const char *name) {
+        _cleanup_free_ void *data = NULL;
+        int r;
+
+        r = read_credential(name, &data, NULL);
+        if (r < 0)
+                return IN_SET(r, -ENXIO, -ENOENT) ? 0 : r;
```

Comment:
i'd really leave this mangling to the caller. maybe some caller might want to default to true rather than false if unset